_i18n/en/resources/moneropedia/unlocktime.md

```diff
@@ -1,14 +1,30 @@
 ---
 terms: ["unlock-time"]
-summary: "a special transaction where the recipient can only spend the funds after a future date, as set by the sender"
+summary: "a special transaction where the recipient(s) can only spend the funds after a future date, as set by the sender"
 ---
 
 {% include disclaimer.html translated="no" translationOutdated="no" %}
 
 ### The Basics
 
-A special transaction where the recipient can only spend the funds after a future date, as set by the sender.
+A special transaction where the recipient(s) can only spend the funds after a future date, as set by the sender.
 
-Unlock time allows you to send a transaction to someone, such that they can not spend it until after a certain number of blocks, or until a certain time.
+Unlock time allows you to send a transaction to someone, such that they (and you if you intend to receive change!) can not spend it until after a certain block height, or until a certain time. These locks are per transaction, not per output. ***Do not unintentionally lock your change outputs.***
```

Comment:
I'm a bit confusing by the wording `Unlock time allows you to send a transaction to someone, such that they (and you if you intend to receive change!) can not spend it until after a certain block height`. Maybe the content of the parenthesis could be moved to before the comma?

Reply:
I also stumbled when reading this sentence, and had to read it at least 3 times to make sense of it hm

Reply:
Yeah if we struggle to understand it, random visitors will be in trouble. Let's change it to something more clear and then get this merged.

Reply:
I removed the parenthesis and added "This means any returning change will be locked too. " to the end of the sentence. Let me know what you think (its basically saying the same thing 3*) 

>Unlock time allows you to send a transaction to someone, such that they can not spend it until after a certain block height, or until a certain time. These locks are per transaction, not per output. This means any returning change will be locked too. ***Do not unintentionally lock your change outputs.***"